docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -17,7 +17,7 @@
   * [Netty SocketIO](https://github.com/mrniko/netty-socketio) 1.x
 * HTTP Client
   * [Feign](https://github.com/OpenFeign/feign) 9.x
-  * [Netflix Spring Cloud Feign](https://github.com/spring-cloud/spring-cloud-netflix/tree/master/spring-cloud-starter-feign) 1.1.x, 1.2.x, 1.3.x
+  * [Netflix Spring Cloud Feign](https://github.com/spring-cloud/spring-cloud-netflix/tree/master/spring-cloud-starter-feign) 1.1.x, 1.2.x, 1.3.x(Except 1.3.3), 1.4.x, 2.x
```

Comment:
Just curious, what happens in 1.3.3?

Reply:
Also here, 2.x should not belong to Netflix right? The old link is not working anymore. Please fix this by the current one.

Reply:
1.1.x needs springboot 1.3.x, 1.2+ needs springboot 1.5.x, 2.x needs springboot 2.x. 
Can't download 1.3.3 dependency in maven repo.

Reply:
If it can't download from the Maven central, we don't need to indicate this in the doc, because users can't use it too.